### PR TITLE
Raw disk access

### PR DESCRIPTION
**Pull request by tomchop**

### CODE REVIEW

File: data/recipes/grr_artifact_grep.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         ["keywords", "Pipe-separated list of keywords to search for (e.g. key1|key2|key3.", null],
         ["--artifacts", "Comma-separated list of artifacts to fetch (override default artifacts).", null],
         ["--extra_artifacts", "Comma-separated list of artifacts to append to the default artifact list.", null],
-        ["--use_tsk", "Use TSK to fetch artifacts.", false],
+        ["--use_raw_filesystem_access", "Use raw disk access to fetch artifacts.", true],
```
Comment (ramo-j) [inline data/recipes/grr_artifact_grep.json:35]: Changing the recipe defaults of raw access from false to true results in different artefact layouts, if memory serves. We're not worried about changing the default?

**Review by ramo-j — APPROVED** [review]
One small question, but otherwise, LGTM.

File: data/recipes/grr_artifact_grep.json (hunk 1 above)
Comment (tomchop) [reply data/recipes/grr_artifact_grep.json:35]: I think the artifacts that we collect remain the same regardless of the access method (if I understood your question correctly)

Comment (ramo-j) [reply data/recipes/grr_artifact_grep.json:35]: Perhaps I'm misremembering then, I thought the grr zip archive output was structured differently.

**Review by ramo-j — APPROVED** [review]

**Pull request merged**